### ISSUE TITLE
Add Array support to ParamValue

### ISSUE THREAD

**Issue by antoinekm** [alii/pathcat#1]
Hey @alii, it could be great to be able to do things like that:

```ts
pathcat("https://example.com", "/:id", {
	id: "123",
	foo: ["b", "a", "r"],
});
// => 'https://example.com/123?foo=b&foo=a&foo=r'
```

**Comment by alii** [comment]
Fixed in #2 (thank you!)
Released as 1.3.0 https://www.npmjs.com/package/pathcat/v/1.3.0

**Comment by alii** [comment]
Also published to jsr.io here - https://jsr.io/@alistair/pathcat@1.3.0
